src/Microsoft.ML.OnnxTransformer/OnnxTransform.cs

```diff
@@ -57,7 +58,7 @@ namespace Microsoft.ML.Transforms.Onnx
     /// Refer to [ONNX](http://onnx.ai) for more information.
     ///
     /// To create this estimator use the following:
-    /// [ApplyOnnxModel](xref:Microsoft.ML.OnnxCatalog.ApplyOnnxModel*)
+    /// [ApplyModel](xref:Microsoft.ML.OnnxCatalog.ApplyOnnxModel*)
```

Comment:
>[ApplyModel](xref:Microsoft.ML.OnnxCatalog.ApplyOnnxModel*) [](start = 8, length = 59)

why we have `*` in the end of ApplyOnnxModel ?
And why it's ApplyModel instead of ApplyOnnxModel?

Reply:
soome slike the * is the equivalent of "all overloads". The link is resolving correctly on smoke-test 

---
In reply to: [278239029](https://github.com/dotnet/machinelearning/pull/3530#discussion_r278239029) [](ancestors = 278239029)